base_local_planner/src/speed_limiters/obstacle_speed_limiter.cpp

```diff
@@ -188,13 +188,7 @@ costmap_2d::ObstructionMsg SpeedLimiter::obstructionToBodyFrame(const costmap_2d
   return out;
 }
 
-void SpeedLimiter::calculateStopDistances()
-{
-  max_distance_to_stop_ = 0.5 * params_.max_linear_velocity_ * params_.max_linear_velocity_ / params_.linear_acceleration_;
-  min_distance_to_stop_ = 0.5 * params_.min_linear_velocity_ * params_.min_linear_velocity_ / params_.linear_acceleration_;
-}
-
-void SpeedLimiter::calculateFootprintBounds(std::vector<geometry_msgs::Point> footprint)
+void ObstacleSpeedLimiter::calculateFootprintBounds(std::vector<geometry_msgs::Point> footprint)
```

Comment:
this should be a reference

Reply:
otherwise you are making an explicit copy of all points in the vector

Reply:
Done